exercises/concept/grade-stats/.docs/introduction.md

```diff
@@ -97,5 +97,6 @@ def add: reduce .[] as $x (null; . + $x);
   | reduce .[] as $elem ([]; [$elem] + .)       # => ["D", "C", "B", "A"]
   ```
 
+[jq-code-add]: https://github.com/stedolan/jq/blob/jq-1.6/src/builtin.jq#L11
```

Comment:
My understanding is that admonitions are parsed as self-contained markdown documents. This change will break the link.

Reply:
This would be fixed in configlet version 4.0.0-beta.10, so make sure you fetch-configlet regularly.